lib/utils/utils.ts

```diff
@@ -155,3 +155,14 @@ export const removeUndefinedProps = (obj: any) => {
 
   return obj;
 };
+
+export const sortOrders = (orders: PeerOrder[], isBuy: boolean) => {
```

Comment:
Minor note but this should probably be `orders: Order[]` since we are not only interested in sorting Peer Orders.